### PR TITLE
316: Redesign login page

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: README/README_JS_CODE_STYLE.md [hunk 1]
```diff
@@ -96,9 +74,7 @@ Use documentation annotation for this.
 ## Angular code style notes
 
 ### Naming
-Module names should start from **mih**
 Controller names should start with uppercase letter and finishes with Ctrl
-Directive name should start from **mih**
 Service name should start from uppercase if it is instantiable value or with lowercase if not and finishes with **Srv**
```
Comment (make-it-happen-team) [inline README/README_JS_CODE_STYLE.md]: I suggest use for services suffix Service - f.e. CalculationService

File: README/README_JS_CODE_STYLE.md [hunk 2]
```diff
@@ -96,9 +74,7 @@ Use documentation annotation for this.
...
 Factory name should start from uppercase if it is instantiable value or with lowercase if not and finishes with **Fct**
```
Comment (make-it-happen-team) [inline README/README_JS_CODE_STYLE.md]: The same with service good practice use only suffix Factory - f.e. ItemFactory

File: README/README_JS_CODE_STYLE.md [hunk 3]
```diff
@@ -1,27 +1,5 @@
 # JS Code style notes
 
-## Tooling
-
-We do use ESLint for code style managing.
```
Comment (make-it-happen-team) [inline README/README_JS_CODE_STYLE.md]: Move this section back

File: README/README_JS_CODE_STYLE.md (hunk 1 above)
Comment (ghost) [reply README/README_JS_CODE_STYLE.md]: Ok

File: README/README_JS_CODE_STYLE.md (hunk 2 above)
Comment (ghost) [reply README/README_JS_CODE_STYLE.md]: Ok

File: README/README_JS_CODE_STYLE.md (hunk 3 above)
Comment (ghost) [reply README/README_JS_CODE_STYLE.md]: We have no ESLint so we don't need this section

Comment (make-it-happen-team) [reply README/README_JS_CODE_STYLE.md]: Good

**Pull request merged**